rest-api/tools/setup_database.sh

```diff
@@ -50,26 +50,22 @@ fi
 
 source tools/setup_vars.sh
 
+# Get a randomly generated password
+function randpw {
+    new_password=$(< /dev/urandom LC_CTYPE=C tr -dc _A-Z-a-z-0-9_\!\@\#\$\%\^\&\*\(\)-+= | head -c${1:-16};echo;)
+    }
+
 # Prompts the user for a database user password.
 function get_required_password {
   db_user_name=$1
-  echo -n "Store and share this password in Valentine with"
-  echo " name=\"$db_user_name\" and description=\"$PROJECT database\"."
-  read -s -p "$db_user_name password for database: " new_password
+  echo "Setting random password for user: $db_user_name "
+  randpw
   echo
   if [ -z "${new_password}" ]
```

Comment:
I don't think you need get_required_password anymore. Just call randpw directly below?